cobalt/site/docs/development/setup-android.md

```diff
@@ -151,81 +155,63 @@ Go ahead and click 'yes' to open the SDK manager to install the following:
     make it through both the shell on your workstation and the shell on the
     device), e.g.:
 
-    ```
+    ```sh
     adb shell am start --esa args --flag_arg,--value_arg=something dev.cobalt.coat
     ```
 
 1.  To monitor log output, watch logcat in another shell with a filter for
     starboard messages:
 
-    ```
+    ```sh
     adb logcat -s starboard:*
     ```
 
 1.  To kill any existing running application process (even if it's no longer the
     active app) use:
 
-    ```
+    ```sh
     adb shell am force-stop dev.cobalt.coat
     ```
 
-## Building/running/debugging (Android Studio IDE)
```

Comment:
@kaidokert @y4vor  how do we feel about removing Android Studio IDE instructions?

Mostly we build via CLI, but I think it is helpful to know how to setup with Android Studio, if necessary. For example: debugging convenience with an IDE and such. I'd be open to leave it out to simplify and we can add back, if needed, but wanted to get some other thoughts here as well.

Reply:
If we can keep them accurate and up to date, i wouldn't remove it. I'm not sure we have been able to, however. It's certainly not "mvp" for this doc

Reply:
Agreed. Give that it's out of date, I'm okay to remove it for now while we take time to review updated guidance for this for C25 LTS Stable and track that effort in a ticket.